feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package com.epmedu.animeal.tabs.more.about
```

Comment:
Please create a sub-package for viewmodel `com.epmedu.animeal.tabs.more.about.viewmodel`